tensorflow_io/core/kernels/kafka_kernels_deprecated.cc

```diff
@@ -242,6 +259,14 @@ class KafkaDatasetOp : public DatasetOpKernel {
                   }
                   out_tensors->emplace_back(std::move(key_tensor));
                 }
+                if (dataset()->message_offset_) {
+                  Tensor offset_tensor(cpu_allocator(), DT_STRING, {});
+                  int64_t offset = message->offset();
+                  int32_t partition = message->partition();
+                  offset_tensor.scalar<tstring>()() =
+                      strings::Printf("%d:%ld", partition, offset);
```

Comment:
Can't we use `std::string` for capturing `partition:offset` like we did for messages or key? (As per lines above)

Reply:
@777ki

Reply:
Ok, I'll go and adjust it， thx，Just a moment